locales/de.json

```diff
@@ -563,7 +563,7 @@
     "focusable-no-name": {
       "pass": "Das Element befindet sich nicht in der Tabreihenfolge und enthält keinen zugänglichen Text.",
       "fail": "Das Element befindet sich in der Tabreihenfolge und enthält keinen zugänglichen Text.",
-      "incomplete": "Es ist nicht möglich herauszufinden ob das Element einen zugänglichen Namen besitzt."
+      "incomplete": "Es ist nicht möglich herauszufinden ob das Zugänglichkeit-Objekt des Elements einen Namen besitzt."
```

Comment:
I don't understand what the `Zugänglichkeit-Objekt des Elements` is supposed to mean. Why did you make these changes? I preferred the old translation.

Reply:
Tried something different there. But this translation doesn't fit everywhere. I think `zugänglicher Name (accessible name)` must be sufficient.